internal/k8s/crdcontroller.go

```diff
@@ -41,6 +41,21 @@ func NewCRDInformers(cs akocrd.Interface) {
 	})
 }
 
+func isHTTPRuleUpdated(oldHTTPRule, newHTTPRule *akov1alpha1.HTTPRule) bool {
+	if oldHTTPRule.ResourceVersion == newHTTPRule.ResourceVersion {
+		return false
+	}
+
+	oldSpecHash := utils.Hash(utils.Stringify(oldHTTPRule.Spec))
+	newSpecHash := utils.Hash(utils.Stringify(newHTTPRule.Spec))
```

Comment:
What if someone changes the order of two properties? That will give a different hash, right?